src/com/sun/ts/tests/signaturetest/README

```diff
@@ -1,431 +1,431 @@
-Overview:
```

Comment:
The github ui and local output from "git show 4a4946e815f0c0425e222c2eee54c2f6f70d1faf", both show that every line of the README was updated.  Could you please check the recent change.  Thanks!

Reply:
Good catch. IDE update making my day interesting :). Fixed now.